e2e/page.ts

```diff
@@ -66,4 +74,40 @@ export default class PlaywrightDevPage {
             setTimeout(() => res(true), ms);
         });
     }
+
+    async openWidget(channelName: string) {
+        const resp = await this.page.request.get(`${baseURL}/api/v4/teams/name/${defaultTeam}/channels/name/${channelName}`);
+        const channel = await resp.json();
+
+        await this.page.goto(`${baseURL}/plugins/${pluginID}/standalone/widget.html?call_id=${channel.id}`);
+        await expect(this.page.locator('#calls-widget')).toBeVisible();
+        await expect(this.page.getByTestId('calls-widget-loading-overlay')).toBeHidden();
+    }
+
+    async hideDocument(hide = true) {
+        await this.page.evaluate((hidden) => {
+            Object.defineProperty(document, 'visibilityState', {value: hidden ? 'hidden' : 'visible', writable: true});
+            Object.defineProperty(document, 'hidden', {value: hidden, writable: true});
+            document.dispatchEvent(new Event('visibilitychange'));
+        }, hide);
+    }
+
+    async expectNotifications(numDesktopNotificationsRejected: number, numDesktopNotificationsSent: number, numNotificationsSounded: number, numNotificationsStoppedAt: number) {
```

Comment:
I understand it gets verbose but looking at `expectNotifications(0,0,0,0`) getting called from someone that hasn't designed all of this feels very confusing :p

Reply:
And here I thought I was doing well by reducing a ton of lines in the test. 😝 
Open to alternatives (no args object...).

Reply:
I won't push on this since it's a test but in app we'd go with the object just so you know :)

Reply:
ehhh... Then I probably would remove it. :P